python/xi_plugin.egg-info/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
```

Comment:
I branched and tested pushing with travis without the egg and it fails. So, I created a setup.py file which will build the egg when the tests are run. So, no  including the .egg isn't necessary.

I did find though that python wasn't building on the osx worker though so I might close this request and open a new one when I figure out that